machine.go

```diff
@@ -36,10 +36,10 @@ type image struct {
 }
 
 type Machine struct {
-	mounts []mountPoint
-	count  int
-	images []image
-	memory int
+	mounts  []mountPoint
```

Comment:
not sure these blank spaces should be in current PR

Reply:
They're in a seperate commit; i guess it could be split out or done directly to master but i'll merge with rebase anyway so little harm